packages/slate-react/src/utils/is-input-data-enter.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * In Android API 26 and 27 we can tell if the input key was pressed by
+ * waiting for the `beforeInput` event and seeing that the last character
+ * of its `data` property is char code `10`.
+ *
+ * Note that at this point it is too late to prevent the event from affecting
+ * the DOM so we use other methods to clean the DOM up after we have detected
+ * the input.
+ *
+ * @param  {String} data
+ * @return {Boolean}
+ */
+
+export default function isInputDataEnter(data) {
```

Comment:
Can you rename this to be Android specific it it's only used there. Or better yet inline this utility functions if it's so custom?

Reply:
Yes, actually, I think I can remove this as there is another function now is-input-data-last-char that does the same thing and more.